source/Tipster.cpp

```diff
@@ -52,6 +55,7 @@ Tipster::Tipster()
 	fURL = new BString("");
 	fArtworkTitle = new BString("");
 	fPreviousTip = new BString("");
+	ind=0;
```

Comment:
Coding style: add spaces before and after "=".
Rename "ind" to something more descriptive.